apps/etterlatte-saksbehandling-ui/client/src/store/reducers/SaksbehandlerGjeldendeOppgaveReducer.ts

```diff
@@ -0,0 +1,13 @@
+import { createAction, createReducer } from '@reduxjs/toolkit'
+
+export const setSaksbehandlerGjeldendeOppgave = createAction<string | null>('saksbehandlerGjeldendeOppgave/set')
+
+const initialState: { saksbehandler: string | null } = {
+  saksbehandler: null,
```

Comment:
Burde denne nullstilles når en oppgave er ferdig? Eller finnes det noen gode måter å gjøre det på?

Reply:
Det gjøres kanskje ikke på noe av redux-staten vår?

Reply:
Jeg skal ikke påberope meg oversikt, men tror ikke det gjøres nei. Denne her går hånd-i-hånd med sjekklista, og der gjøres det i allefall ikke.

Jeg tror ikke det er noe problem her så lenge reduceren brukes kun innenfor behandlingsflyt iallefall, for man oppdaterer når man åpne den (dvs sidemenyen).